### ISSUE TITLE
support for ssl / scram-512 connections to Kafka

### ISSUE THREAD

**Issue by lzwaan** [probably-nothing-labs/denormalized#62]
Hi, I know this project is in a early stage, but is it possible to connect to kafka via SSL?

**Comment by emgeee** [comment]
Not yet but it shouldn't be too hard to implement. Is that a feature you'd be interested in?

**Comment by rxraghu** [comment]
Yes, we also use KAFKA_SECURITY_PROTOCOL as sasl_ssl so ssl support would be required. I could try to use SASL_PLAINTEXT for development, but I can't find a way to set the SA username and password (for the broker) in `Context().from_topic`. Do we need to set some env variables?
